src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -266,7 +269,7 @@ export const Browser: FunctionComponent<BrowserProps> = (
   }));
 
   const getBrowserWidth = (): string => {
-    if (props.drawerOpened) {
+    if (isDrawerOpened) {
       return 'calc(41px + 0vw)';
```

Comment:
0 is always 0; doesn't matter whether in pixels or in vws :smile:

Reply:
TypeScript does not infer the return type as string properly and that's why I explicitly added `string` as the return type there.

Reply:
Sorry. What I meant here was that the `'calc(41px + 0vw)';` string doesn't make sense on several levels:
- `0vw` is the same as just 0
- 41px + 0 is 41 px
- therefore, calc is unnecessary

Reply:
Ah I get what you mean. I'll fix that.

Reply:
Unfortunately the fix for this breaks the `react-spring` animation. So I had to put the same value back. I found out from @msmanoj, who worked on this animation, that `react-spring` requires all widths to be in the same format.

Reply:
Could you please add a comment (something like `calc and 0vw needed to preserve same format across different ReactSpring values`, so that we don't get confused the next time?

Reply:
@azangru I have added a comment in d15edff